src/components/Capture/Document.tsx

```diff
@@ -229,7 +229,7 @@ class Document extends Component<Props> {
         <DocumentLiveCapture
           containerClassName={style.liveDocumentContainer}
           documentType={documentType}
-          isUploadFallbackDisabled={!uploadFallback}
+          isUploadFallbackDisabled={uploadFallback ? !uploadFallback : false}
```

Comment:
This statement always returns `false` 🤔

Reply:
Ah, you're right. & there's a simpler way of doing what I'm trying to achieve here. Will fix.